src/core/AutoRest.Core/Settings.cs

```diff
@@ -73,8 +73,6 @@ public Settings()
             ValidationLevel = Category.Error;
             ModelsName = "Models";
             CodeGenerationMode = "rest-client";
-
-            CustomSettings.Add("ClientSideValidation", true);
```

Comment:
handled below instead (`src/core/AutoRest/Program.cs`), unfortunately the legacy CLI parsing mechanism overwrites `CustomSettings` completely, such that this line had no effect.